File: golang/problem-001-a-simple.go
package main

import (
	"fmt"
)

func main() {
	ceiling := 1000
	sum := 0

	for i := 0; i < ceiling; i++ {
		if i%3 == 0 || i%5 == 0 {
			sum += i
		}
	}

	fmt.Println("Total sum is", sum)
}

File: ruby/problem-001-a-simple.rb
ceiling = 1000

sum_of_series =
  (1...ceiling).select do |i|
    i % 3 == 0 || i % 5 == 0
  end.inject(0) do |sum, i|
    sum += i
  end

puts "The sum of multiples of 3 or 5 up to #{ceiling} is #{sum_of_series}"

